config/environments/production.rb

```diff
@@ -39,21 +44,31 @@
                     "https://#{ENV.fetch('HEROKU_APP_NAME', '')}.herokuapp.com")
   }
 
+  # Assume all access to the app is happening through a SSL-terminating reverse proxy.
+  # Can be used together with config.force_ssl for Strict-Transport-Security and secure cookies.
+  # config.assume_ssl = true
+
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  # config.force_ssl = true
+  config.force_ssl = true
```

Comment:
I wonder whether there's a reason this wasn't previously set? It might be worth leaving it commented out.

Reply:
Addressed in 33c3b1c